### PR TITLE
Ported Content Template for Developers article from wiki. #685

### PR DESCRIPTION
**Pull request by silvia-odwyer**
Hi there, 
I have ported the [article](http://wiki.sensenet.com/Content_Template_-_for_Developers) from the wiki and it is now available as a Markdown file. If you would like me to make further edits, just let me know, and I can commit those. 

This references Issue #685 and hopefully should close it.

### CODE REVIEW

File: docs/content-template-for-developers.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+var templateNode = template.FirstOrDefault();
+```
+
+## Related links 
+[Content - for Developers](http://wiki.sensenet.com/Content_-_for_Developers)
+[Node - for Developers](http://wiki.sensenet.com/Node_-_for_Developers)
+[Getting started - developing applications](http://wiki.sensenet.com/Getting_started_-_developing_applications)
+
+## References
+There are no related external articles for this article.
```
Comment (herflis) [inline docs/content-template-for-developers.md]: This part could be simply removed till the referenced docs are not ported to github.

**Comment by herflis** [comment]
Hi @silvia-odwyer, 
thank you for your awesome work! 🙌 I've suggested one minor thing to change but it does not affect that you'll receive some extra goodies from us if you fill out [this form](https://www.sensenet.com/Hacktoberfest2019) 👕+💚

File: docs/content-template-for-developers.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
+# Overview
```
Comment (herflis) [inline docs/content-template-for-developers.md]: Add the header part as:

---
title:  "Content Template for Developers "
source_url: 'https://github.com/SenseNet/sensenet/blob/master/docs/content-template-for-developers.md'
category: Developers
version: v6.0
tags: [content, content template]
---

**Comment by silvia-odwyer** [comment]
Hi Aniko, 
I've made the requested changes by adding the header and removing the two sections referenced, thanks so much for reviewing! 😄

**Comment by herflis** [comment]
Hey @silvia-odwyer ,
did you register [here](https://sensenet.com/Hacktoberfest2019)? I cannot see your name on our list :)

**Pull request merged**